libs/beacon/src/beacon_setup_service.cpp

```diff
@@ -1088,7 +1086,7 @@ void BeaconSetupService::OnNewShares(MuddleAddress
   bool in_cabinet{false};
   for (auto &m : beacon_->aeon.members)
   {
-    if (m.identifier() == from)
+    if (m == from)
```

Comment:
best to rename m here to member or the like - single letter variable not our style

Reply:
Ok, will do.